src/pages/Merchant/Register/Register.js

```diff
@@ -0,0 +1,180 @@
+import React, { useState, useRef } from "react";
+import { useNavigate } from "react-router-dom";
+
+import styles from "./Register.module.scss";
+
+import Button from "components/Button/Button";
+import InputControl from "components/InputControl/InputControl";
+
+function Register() {
+  const [errors, setErrors] = useState({});
+
+  const fnameRef = useRef();
+  const lnameRef = useRef();
+  const emailRef = useRef();
+  const mobileRef = useRef();
+  const passRef = useRef();
+  const confirmpassRef = useRef();
+
+  const validateForm = () => {
+    const dummyErrors = {};
+    if (fnameRef.current.value === "") {
+      dummyErrors.fname = "Enter first name";
+      setErrors(dummyErrors);
+    }
+
+    if (lnameRef.current.value === "") {
+      dummyErrors.lname = "Enter last name";
+      setErrors(dummyErrors);
+    }
+
+    if (mobileRef.current.value === "") {
+      dummyErrors.mobile = "Enter mobile number";
+      setErrors(dummyErrors);
+    } else {
+      if (
```

Comment:
same mistake as in login page validateForm

Reply:
resolved